### ISSUE TITLE
Example training of quora_duplicate_questions : Test Dataset Evaluation  

### ISSUE THREAD

**Issue by working12** [UKPLab/sentence-transformers#685]
In this example script [link](https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/quora_duplicate_questions/training_OnlineContrastiveLoss.py), we can see two splits, one is the training dataset (train pairs) and another portion is for validation (dev_pairs). Usually, in the pipeline, we sometimes use the final test dataset. (I have a test pairs portion as well over which I need to run the final trained model).

So, in case we need to run it at the end of the training and check it, how can we do it? Do we need to create an evaluator for the test dataset and append it to the `evaluators`? (Just like the 3 dev pairs evaluator in the example) 

Thanks.

**Comment by nreimers** [comment]
Hi @working12 
Yes, you can use again the evaluator on your test set. Or you implement your own code to test the performance on the test set.

For an example that uses an evaluator class, see:
https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/sts/training_stsbenchmark.py

**Comment by working12** [comment]
@nreimers Thanks. You are the best.
